css/blockonomics_options.css

```diff
@@ -0,0 +1,133 @@
+h1{
+    padding-bottom: 7px !important;
+}
+
+.nav-tab {
+    cursor: pointer !important;
+}
+
+h2 {
+    margin-block-end: 4px !important;
+}
+
+h3 {
+    margin-block-start: 0px !important;
+    margin-block-end: 0px !important;
+}
+
+h4 {
+    margin-block-start: 10px !important;
+    margin-block-end: 2px !important;
+    font-size: medium;
+}
+
+label {
+    cursor: auto;
+}
+
+.bnomics-default-cursor {
+    cursor: default !important;
+}
+
+.wp-core-ui .notice.is-dismissible {
+    display: none;
+}
+
+
+.blockonomics-narrow-th {
+    width: 90px !important;
+}
+
+.form-table td p {
+    margin-top: 0px !important;
+}
+
+.bnomics-width {
+    max-width: 550px !important;
+}
+
+.bnomics-width-withdraw {
+    max-width: 680px !important;
+}
+
+
+.form-table th {
+    padding: 5px 10px 5px 0;
+}
+
+.bnomics-options-input {
+    max-width: 100%; 
+}
+
+.bnomics-options-callback-icon {
+    font:400 24px/1 dashicons;
+    margin-left: 3px;
+    top: 6px;
+    position:relative;
+    text-decoration: none;
+}
+
+.bnomics-options-bold {
+    font-weight:bold;
+}
+
+
+.bnomics-options-intendation-heading {
+    margin-left:40px; 
+    padding: 0px 0px !important;
+}
+
+
+.bnomics-options-green-check-mark {
+    font:1000 25px/1 dashicons; 
+    top:0px; 
+    color:green;
+}
+
+.bnomics-options-no-padding {
+    padding: 0px 0px !important;
+}
+
+.bnomics-options-no-margin {
+    margin: 0px 0px !important;
+}
+
+
+.bnomics-options-red-cross-mark {
+    font:1000 25px/1 dashicons; 
+    top:0px; 
+    color:red
+}
+
+
+.bnomics-options-small-intendation {
+    margin-left:12px !important;
```

Comment:
The Settings Tab and Currencies Tab have different left margins, which appear to be unnecessary.

Reply:
I removed much of the left margins in the Currencies Tab. Let me know what you think.